src/locales/source/en.json

```diff
@@ -90,7 +90,6 @@
     "less-than": "Less than a minute",
     "material": "Material",
     "na": "NA",
-    "est-pnt-time": "Est. Printing Time",
```

Comment:
This is a new label because when list the files there is an estimation  comming from slicer, what is different when open the project in preview because it is calculeted.